.ci/updatecli/values.d/scm.yml

```diff
@@ -0,0 +1,5 @@
+scm:
+  enabled: true
+  owner: elastic
+  repository: logstash
+  commitusingapi: true
```

Comment:
`nit`: this is hardcoded in the updatecli manifest

```suggestion
```

Reply:
updated